src/Synergism.ts

```diff
@@ -1790,12 +1790,18 @@ export const format = (
     }
 
     // If the power is less than 12 it's effectively 0
-    if (power < -12) {
+    if (power < -15) {
         return '0';
     }
 
     // If the power is negative, then we will want to address that separately.
     if (power < 0 && !isDecimal(input) && fractional) {
+        if (power <= -15) {
+            return `${format(mantissa, accuracy, long)} / ${Math.pow(10, -power - 15)}Qa`
+        }
+        if (power <= -12) {
+            return `${format(mantissa, accuracy, long)} / ${Math.pow(10, -power - 12)}T`
+        }
```

Comment:
?

Reply:
this is a live change now, was awwhy's bug fix for global speed not showing negative exponents. without it I had the same bug in the history tab